### PR TITLE
[no-issue] clean e2e test PS

### PR DESCRIPTION
**Pull request by eromano**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [x] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: e2e/pages/adf/contentServicesPage.js [hunk 1]
```diff
@@ -397,7 +397,6 @@ var ContentServicesPage = function () {
         var deferred = protractor.promise.defer();
         errorSnackBar.getText().then(function (text) {
             /*tslint:disable-next-line*/
```
Comment (cristinaj) [inline e2e/pages/adf/contentServicesPage.js]: Is this line needed?

File: e2e/pages/adf/core/toggleState.js [hunk 2]
```diff
@@ -19,32 +19,23 @@ var Util = require('../../../util/util');
...
-            Util.waitUntilElementIsClickable(toggle);
-            if (value.indexOf('mat-checked')===-1) {
+        toggle.getAttribute('class').then((check) => {
+            if (check.indexOf('mat-checked') < 0) {
                 toggle.click();
             }
         });
-        return this;
-    };
+    }
```
Comment (cristinaj) [inline e2e/pages/adf/core/toggleState.js]: ; missing

File: e2e/pages/adf/core/toggleState.js [hunk 3]
```diff
@@ -19,32 +19,23 @@ var Util = require('../../../util/util');
...
-            if (value.indexOf('mat-checked')!==-1) {
-                finalToggleButton.click();
+        toggle.getAttribute('class').then((check) => {
+            if (check.indexOf('mat-checked') >= 0) {
+                toggle.click();
             }
         });
-        return this;
-    };
+    }
```
Comment (cristinaj) [inline e2e/pages/adf/core/toggleState.js]: ; missing

File: e2e/pages/adf/core/toggleState.js (hunk 2 above)
Comment (eromano) [reply e2e/pages/adf/core/toggleState.js]: 👍

File: e2e/pages/adf/core/toggleState.js (hunk 3 above)
Comment (eromano) [reply e2e/pages/adf/core/toggleState.js]: 👍

**Pull request merged**